src/plugin/render_operations/renderer_copy_operation.cpp

```diff
@@ -17,6 +17,7 @@ namespace wmr
 		, m_renderer(dynamic_cast<const ViewportRendererOverride*>(MHWRender::MRenderer::theRenderer()->findRenderOverride(settings::VIEWPORT_OVERRIDE_NAME))->GetRenderer())
 		, m_blit_operation(blit_operation)
 	{
+		LOG("Attempting to get a reference to the renderer.");
```

Comment:
It actually already tried to get a reference when this `LOG` is called 😉 Doesn't matter anyway

Reply:
👍